robot/basestation/templates/elements/navigation/GPS.html

```diff
@@ -1,7 +1,4 @@
-<h5 class="container-header">GPS Map</h5>
-<div class="imageWrapper">
-  <img src="../static/img/gps_map.jpg" class="mx-auto d-block" id="arm-position"/>
-  <span class="overlayGpsDot" style="top:50px; left:100px; background-color:blue">Rover</span>
-  <span class="overlayGpsDot" style="top:70px; left:20px; background-color:red">Antenna</span>
-  <span class="overlayGpsDot" style="top:150px; left:300px;">Goal</span>
+<h5 class="container-header">Map</h5>
+<div id="canvasContainer">
+  <script src='../../../static/js/elements/navigation/localization.js'></script>
```

Comment:
would it be possible to import to script in the rover page just like all the scripts are imported?

Reply:
most likely